pkg/integration/testdata/exclude.json

```diff
@@ -1,5 +1,6 @@
 [
   "/swagger/*any",
+  "/ns/lookup",
```

Comment:
any reason for exclude? can we add integration testing for ns?

Reply:
currently integration tests only under specific platform, @Pantani may have more insights